react/src/App.jsx

```diff
@@ -0,0 +1,302 @@
+    window.helpshiftConfig.userId = "captain_planet12";
+    window.helpshiftConfig.userEmail = "captain@example.com";
+    onUpdateHelpshiftConfig();
+    setIsLoggedIn(true);
+  };
+
+  const onHandleLogout = () => {
+    window.helpshiftConfig.userId = "";
+    window.helpshiftConfig.userEmail = "";
+    onUpdateHelpshiftConfig();
+    setIsLoggedIn(false);
+  };
+
+  const onHandleLauncherChange = (e) => {
+    const val = e.target.value;
+    if (val == "showLauncher") {
+      window.Helpshift("show");
+      setSelectedLauncherOption("showLauncher");
+    } else {
+      window.Helpshift("hide");
+      setSelectedLauncherOption("hideLauncher");
+    }
+  };
+
+  const onChangeWidgetPosition = (e) => {
+    const value = e.target.value;
+    setPosition(value);
+  };
+
+  const onWidgetPositionApply = () => {
+    window.helpshiftConfig.widgetOptions.position = position;
+    onUpdateHelpshiftConfig();
+  };
+
+  const onHandleFullScreenChange = (e) => {
+    const val = e.target.value;
+    setSelectedFullScreenOption(val);
+    if (val === "exitFullScreen") {
+      window.helpshiftConfig.widgetOptions.fullScreen = false;
+    } else {
+      window.helpshiftConfig.widgetOptions.fullScreen = true;
+    }
+
+    onUpdateHelpshiftConfig();
+  };
+
+  useEffect(() => {
+    const newUnreadMessagesEventHandler = function (data) {
+      setMessageUnreadCount(data.unreadCount);
+    };
+
+    window.Helpshift(
+      "addEventListener",
+      "newUnreadMessages",
+      newUnreadMessagesEventHandler
+    );
+
+    return () => {
+      window.Helpshift(
+        "removeEventListener",
+        "newUnreadMessages",
+        newUnreadMessagesEventHandler
+      );
+    };
+  }, []);
+
+  const messageAddEventHandler = useCallback((data) => {
+    setMessage(data.body);
+  }, []);
+
+  const onAddMessageEventClick = () => {
+    window.Helpshift("addEventListener", "messageAdd", messageAddEventHandler);
+    setmessageEventListenerIsAdded(true);
+  };
+
+  const onRemoveMessageEventClick = () => {
+    window.Helpshift(
+      "removeEventListener",
+      "messageAdd",
+      messageAddEventHandler
+    );
+    setmessageEventListenerIsAdded(false);
+    setMessage("");
+  };
+
+  return (
+    <div className="max-w-3xl">
+      <h1 className="text-5xl text-center">Web Chat</h1>
+
+      <div className="w-full border-gray-300 mt-12 rounded-lg">
+        <h3 className="py-2 pl-3 text-lg rounded-t-lg font-bold uppercase text-gray-800">
+          Quick setup
+        </h3>
+        <div className="m-3">
+          <div className="mt-2 flex justify-between">
+            <h4 className="text-lg py-2">Update config</h4>
+            <div>
+              <button onClick={onUpdateHelpshiftConfig}>
+                Update Helpshift Config
+              </button>
+            </div>
+          </div>
+          <div className="mt-2 flex justify-between">
+            <h4 className="text-lg py-2">Login with helpshift</h4>
```

Comment:
Change this text to `Login user`.
Helpshift doesn't offer any auth solution, so this is misleading.